crowdsec-docs/docs/getting_started/intro.md

```diff
@@ -18,6 +18,10 @@ See [Version Matrix](/getting_started/versions_matrix.md) for a list of supporte
 
 ## Why is my Security Engine classed as a "log processor" within the console?
 
-The `Security Engine` comes compiled with a number of optional features that can be enabled or disabled at runtime. One of these features is called the "LAPI" (Local API). If this feature is disabled at runtime, the Security Engine will be classed as a "log processor" within the console as it will only be able to process logs and forward the alerts to the local API you define in the configuration.
+The `Security Engine` comes compiled with a number of core components that can be enabled or disabled at runtime.
+
+One of these features is called the "LAPI" (Local API). If this feature is disabled at runtime, the Security Engine will be classed as a "log processor" within the console as it will only be able to process logs and forward the alerts to the local API you define in the configuration.
```

Comment:
I'm not a fan of this angle.  
Do we actually want to say that a SE becomes or is classes as a log processor ? is this not just confusing for reader.
In the config don't we just say that the LAPI is remote hence only the LP will run on this SE? but I don't think we need to explicitylcall this SE a "log processor" it would only serve our old schemas where we put that naming.

If we talk about a distributed setup VS an autonomous setup, it's just multiple SE talking to a chosen central one

@buixor @blotus ? what do you think ?

Reply:
TL;DR;

I'd rather say:   

The SE can run in autonomous mode where a single Security Engine is use, but for some reasons (scaling or perimeter segmentation) you might want to have a distributed setup where multiple SE process logs and send alerts to a central one via LAPI